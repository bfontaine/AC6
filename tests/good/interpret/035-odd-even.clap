def even (x) = if (x = 0) then True else if (x = 1) then False else odd (x - 1)
with odd (x) = if (x = 0) then False else if (x = 1) then True else even (x - 1)

val z = even 42

