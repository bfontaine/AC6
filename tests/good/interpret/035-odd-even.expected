:- even = <code>
:- odd = <code>
:- z = True
